### PR TITLE
Rename RapidReadyForDecision to ClaimsFastTracking and remove dead code

### PR DESCRIPTION
**Pull request by nanotone**
## Summary
- This is a minor book-keeping PR that renames a module and removes dead code. There is no behavior change.
- Because this rename actually reflects a change in team organization, it's worth being explicit here about code stewardship. The Rapid Ready for Decision (RRD) project, now in skeletal maintenance mode, was the first project under what is now called the Benefits Portfolio's Claim Fast Tracking crew ([#benefits-cft](https://dsva.slack.com/archives/C01AXBZA3QR)). This part of the codebase is owned by the two teams in that crew:
  - Contention Classification ([#benefits-cft-classification](https://dsva.slack.com/archives/C04AZ8T7XN1))
  - Employee Experience ([#benefits-cft-employee-exp](https://dsva.slack.com/archives/C01BVF2A3V0))

## Related issue(s)
- This PR lays the groundwork for department-of-veterans-affairs/abd-vro#1804

## Testing done
- All tests still pass.

## What areas of the site does it impact?
No behavior change.

### CODE REVIEW

File: app/models/concerns/form526_claim_fast_tracking_concern.rb [hunk 1]
```diff
@@ -85,7 +84,7 @@ def increase_only?
   end
 
   def diagnostic_codes
-    disabilities.map { |disability| disability['diagnosticCode'] }
+    disabilities.pluck('diagnosticCode')
```
Comment (nanotone) [inline app/models/concerns/form526_claim_fast_tracking_concern.rb:87]: Required by a recently added Rubocop rule [MapToHash](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/MapToHash)

File: app/services/claim_fast_tracking/constants.rb [hunk 2]
```diff
@@ -13,7 +13,7 @@ class Constants
       }
     }.freeze
 
-    DISABILITIES_BY_CODE = DISABILITIES.map { |k, v| [v[:code], k] }.to_h
+    DISABILITIES_BY_CODE = DISABILITIES.to_h { |k, v| [v[:code], k] }
```
Comment (nanotone) [inline app/services/claim_fast_tracking/constants.rb:16]: Required by recently added Rubocop warning [Style/MapToHash](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/MapToHash)

File: app/models/concerns/form526_claim_fast_tracking_concern.rb [hunk 3]
```diff
@@ -6,11 +6,10 @@
 
 # rubocop:disable Metrics/ModuleLength
 # For use with Form526Submission
-# TODO rename Form526RapidReadyForDecisionConcern to Form526ClaimFastTrackingConcern
-module Form526RapidReadyForDecisionConcern
+module Form526ClaimFastTrackingConcern
```
[the remark this answers is not in the document]
Comment (lukey-luke) [reply app/models/concerns/form526_claim_fast_tracking_concern.rb:9]: Thanks for handling tech debt here @nanotone !!

**Review by lukey-luke — APPROVED** [review]
👍 Thanks for doing this. Looks good to me.

**Review by rmtolmach — APPROVED** [review]

**Pull request merged**